flask_cors/__init__.py

```diff
@@ -238,17 +244,25 @@ def cors_after_request(resp):
                 The actual after-request handler, retains references to the
                 the options, and definitions of resources through a closure.
             '''
+            logger.debug('Checking CORS for request')

```

Comment:
I think this is overly verbose. I think as a goal, we should only log stuff when we know we are doing work, and only when it will be somewhat unique to the request.